src/main/java/appeng/parts/reporting/PartInterfaceTerminal.java

```diff
@@ -33,6 +33,10 @@
 	private static final CableBusTextures FRONT_DARK_ICON = CableBusTextures.PartInterfaceTerm_Dark;
 	private static final CableBusTextures FRONT_COLORED_ICON = CableBusTextures.PartInterfaceTerm_Colored;
 
+	public String in = "";
+	public String out = "";
+	public boolean onlyInterfacesWithFreeSlots = false;
```

Comment:
also save status of MA button

Reply:
What's the point, interface terminal GUI has no sub-GUIs, so container cannot outlive the GUI. I'm actually tempted to remove those strings from the part

Reply:
Let's leave this conversation opened for future implementation, but for now it's okay.

Reply:
Sure. (To make it actually persistable it would have to be synchronized wtih server and saved to NBT)